doc/data/messages/s/simplifiable-if-expression/good.py

```diff
@@ -0,0 +1,5 @@
+FLYING_THINGS = ["bird", "plane", "superman", "this example"]
+
+def is_flying_animal(an_object):
+    is_flying = isinstance(an_object, Animal) and an_object.name in FLYING_THINGS
+    return is_flying, not is_flying
```

Comment:
```suggestion
    is_flying = isinstance(an_object, Animal) and an_object.name in FLYING_THINGS
    is_not_flying = not is_flying
    return is_flying, is_not_flying
```

Perhaps? To show how to solve this when assigning?